src/test/java/io/confluent/idesidecar/restapi/util/CPDemoTestEnvironment.java

```diff
@@ -77,18 +73,15 @@ public void start() {
       registerRootCA();
```

Comment:
nit: zookeeper also mentioned in L66 above

Reply:
all good, not a nit, thanks for catching!